packages/aws-cdk-lib/aws-lambda-nodejs/lib/types.ts

```diff
@@ -221,9 +221,22 @@ export interface BundlingOptions extends DockerRunOptions {
    * });
    * ```
    *
+   * If an argument may be specified multiple times, an array of values may be provided:
+   *
+   * ```ts
```

Comment:
based on the above comment, this should be `text` not `ts` too.

Reply:
changed 👍